config/install/apigee_edge.developer_app_settings.yml

```diff
@@ -1,6 +1,6 @@
 langcode: en
-entity_label_singular: ''
-entity_label_plural: ''
+entity_label_singular: 'App'
```

Comment:
> Leave empty to use the default "Developer App" label.'
> Leave empty to use the default "Developer Apps" label.'

These messages should be changed in `\Drupal\apigee_edge\Form\DeveloperAppAliasForm::buildForm()` as well.
Instead of simply replacing the current "Developer app" and "Developer apps" string with "App" or "Apps", placeholders should be used, `@developer_app`  and `@developer_apps` respectively. Value of `@developer_app` should be the *non-overridden* value of `entity_label_singular` from the `apigee_edge.developer_app_settings` config object _if it is set_ otherwise it should fallback to the developer app entity's default entity label "Developer app" (`Drupal\apigee_edge\Entity\DeveloperApp`). The same trick should be used for the `@developer_apps` placeholder.

Reply:
Well, I have just realized that this is not going to work the way like I described so let's leave it as it is. Entity label in `Drupal\apigee_edge\Entity\DeveloperApp` should not be overridden, so what we leave it as it is. We provide a "default override" for developer app labels that can be removed and in that case the system falls back to what it is set in the annotations.